uberduck_ml_dev/vocoders/hifigan.py

```diff
@@ -0,0 +1,494 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/vocoders.hifigan.ipynb (unless otherwise specified).
+
+__all__ = ['HiFiGan', 'ResBlock1', 'ResBlock2', 'Generator', 'DiscriminatorP', 'MultiPeriodDiscriminator',
+           'DiscriminatorS', 'MultiScaleDiscriminator', 'feature_loss', 'discriminator_loss', 'generator_loss',
+           'LRELU_SLOPE', 'AttrDict', 'build_env', 'init_weights', 'apply_weight_norm', 'get_padding']
+
+# Cell
+
+""" from https://github.com/jik876/hifi-gan """
+
+import argparse
+import json
+import datetime as dt
+import numpy as np
+from scipy.io.wavfile import write
+
+import torch
+import torch.nn.functional as F
+import torch.nn as nn
+from torch.nn import Conv1d, ConvTranspose1d, AvgPool1d, Conv2d
+from torch.nn.utils import weight_norm, remove_weight_norm, spectral_norm
+
+# Cell
+
+
+class HiFiGan:
```

Comment:
This is the only addition

Reply:
Thoughts on calling it `HifiGanGenerator`, since it looks like this class only deals with the GAN generator and not the discriminator?

Reply:
Yup works for me